### ISSUE TITLE
Question: Why doesn't flipper support "use_frameworks"?

### ISSUE THREAD

**Issue by meftunca** [facebook/flipper#4855]
Why doesn't flipper support "use_frameworks"? As developers, unfortunately, we cannot use flipper together with many libraries like "firebase". Please provide support on this.

Also, if you list the technologies you use in your desktop application, we can provide support for tauri, which uses less memory than electron.

Finally, I understand that maintaining a stable structure like React Native can be challenging. It is evident that a robust debugger like Flipper offers significant advantages. However, performance issues can be quite frustrating, particularly on less powerful computers. The more we optimize the utilization of RAM and CPU, the more comfortable development experience we can provide

**Comment by pavelustenko** [comment]
I completely agree. I'm using react-native-mmkv which doesn't support Chrome debugger and suggests using Flipper as debug tool for my project. One of the next steps should be integrating with Firebase Cloud Messaging, but it requires use_frameworks to work, so I have to use another debugger if one exists. This is very frustrating.

**Comment by fendorio** [comment]
I am in the exact same boat on a project I'm working on. Using react-native-mmkv, want to use firebase but am now blocked and need to either find a new debugger or use an alternative to Firebase. This is the one project where I'm tied to Flipper, and it's the only project where the debugger has caused more headaches than the project itself!

**Comment by fatlinesofcode** [comment]
+1 we'd stuck using an old version of react-native-firebase v14, as newer version require use_frameworks

**Comment by atultiwaree** [comment]
Sadly no updates!!
